lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -296,11 +309,16 @@ def createTransformAttributesSection(self, sectionName, attrsToAdd):
         geomX = UsdGeom.Xformable(self.prim)
         xformOps = geomX.GetOrderedXformOps()
         xformOpOrderNames = [op.GetOpName() for op in xformOps]
-        xformOpOrderNames.append(UsdGeom.Tokens.xformOpOrder)
+
+        if self.showArrayAttributes:
+            xformOpOrderNames.append(UsdGeom.Tokens.xformOpOrder)
```

Comment:
Are you sure about this? Even though attribute is an array type, I think we always want to show not matter what the value of the optionVar is.

Reply:
According the spec, the token[] should be hidden. I will double check with Nat if we want a exception for this one

Reply:
Ok I spoke with Nat and xformOpOrder will be an exception